backend/apps/github/models/issue.py

```diff
@@ -180,12 +178,6 @@ def bulk_save(issues, fields=None):
         """Bulk save issues."""
         BulkSaveModel.bulk_save(Issue, issues, fields=fields)
 
-    @staticmethod
-    @lru_cache
-    def open_issues_count():
-        """Return open issues count."""
-        return Issue.open_issues.assignable.count()
-
```

Comment:
As you removed existing method you run tests to check it still works.
The problem is that Nest isn't fully covered with tests. NestBot still uses these methods and need to be updated too. So here you have 2 options:
  - keep the old methods and their signatures while updating their logic with `IndexBase` stuff
  - replace old methods with `IndexBase.get_total_count` everywhere by using search